api/mailer/SceGoogleApiHandler.js

```diff
@@ -147,6 +147,34 @@ class SceGoogleApiHandler {
       }
     });
   }
-}
 
-module.exports = { SceGoogleApiHandler };
+  /**
+   * Grabs all the events for a given calendar by its id.
+   * will log the next 10 events into the console.
+   * @param calendarId {string} calendar id for which calendar to pull from
+   */
+  getEventsFromCalendar(calendarId) {
+    // creates a const calendar for use in this method
+    const calendar =
+      google.calendar({ version: 'v3', auth: this.oAuth2Client });
+    // lists the next 10 events in the current calendar
+    calendar.events.list({
+      calendarId: calendarId,
+      timeMin: (new Date()).toISOString(),
+      maxResults: 10,
+      singleEvents: true,
+      orderBy: 'startTime',
+    }, (err, res) => {
+      if (err) return ('The API returned an error: ' + err);
+      const events = res.data.items;
+      if (events.length) {
```

Comment:
while i think it is good to alert the user if no events were found, let's just return the `events` array regardless if its empty or not. e.g.

```js
const events = res.data.items;
events = events.map((event, i) => {
  const start = event.start.dateTime || event.start.date;
  return (`${start} - ${event.summary}`);
});
return events;
```